Automattic/wp-calypso
# Cancellation Flow: Tweak Cancellation Reasons

#### Changes proposed in this Pull Request

* As a part of the solution-oriented cancellation flow project pcbrnV-3O0-p2,  this PR tweaks the cancellation reasons we show in the cancellation flow, so they are categorized into main/subgroups.

#### Testing instructions



* Try to cancel the subscription of your paid plan.
  <img width="957" alt="Cancel_Purchase_‹_Premium_Monthly_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/153840409-42051ef8-7016-486b-b364-4221b2915d6d.png">
* On the "Share your feedback" page, you will see the new cancellation reason options in the two selectboxes. The full list of the options can be found on pcbrnV-3O0-p2.
  <img width="800" alt="Cancel_Purchase_‹_Premium_Monthly_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/153843531-e256e01b-1be8-470b-84c7-18c7043586a2.png">
* In addition to the two select boxes, there should be some upsell nudges for specific cases.
  - Lower plan than Buisness Plan, "Can’t use a plugin/theme" => Upgrade to atomic upsell
  - Business plan, "Can’t use a plugin/theme" => Upgrade to business atomic upsell
  - Premium Plan, "Too expensive" or "I want a cheaper plan" => Downgrade to Personal plan upsell
  - An yearly/bi-yearly plan, "Too expensive" or "I want a cheaper plan" => Downgrade to a corresponding monthly plan
  - A monthly plan on a site that has never consumed the free offer, "I don’t have time." or "My site is not ready" => Free month offer
